.github/workflows/actions.yml

```diff
@@ -0,0 +1,68 @@
+name: documentation
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'website/**'
+      - 'docs/**'
+
+jobs:
+  docs-setup:
+    if: github.event_name != 'push'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+      - name: Setup NodeJS
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Run Test Build
+        run: |
+          cd website
+          if [ -e yarn.lock ]; then
+          yarn install --frozen-lockfile
+          elif [ -e package-lock.json ]; then
+          npm ci
+          else
+          npm i
+          fi
```

Comment:
Given that we use NPM there is not really a need to check for yarn etc. 

```sh
cd website
npm install
```

should be sufficient here.